core/interfaces/rebase.py

```diff
@@ -301,13 +302,21 @@ def _get_diverged_outside_rebase(self):
     def render_conflicts_bindings(self):
         return self.conflicts_keybindings if self._in_rebase else ""
 
-    def base_ref(self):
+    def base_ref(self, reset_ref=False):
         base_ref = self.view.settings().get("git_savvy.rebase.base_ref")
 
-        if not base_ref:
+        if not base_ref or reset_ref:
             project_data = sublime.active_window().project_data() or {}
             project_settings = project_data.get('settings', {})
-            base_ref = project_settings.get("rebase_default_base_ref", "master")
+            base_ref = project_settings.get("rebase_default_base_ref")
```

Comment:
This is irrelevant to this PR, but I would also like to discuss about this setting `rebase_default_base_ref`. It seems it is the only place we use a project setting while we are using `git config` to set GitHub integration repo and branch.

I just want to make sure there will be a consistent way for storing user settings. So which one shall we use if there is such a need? A: `git config` or B: sublime text project settings.

Reply:
I personally don't think "sublime-specific settings" belong in `git config` ...

My project-settings files are synced to dropbox, and I'd like to avoid defining the override ref again for every new project when I switch machines. Then again, this feature kind of makes this a non-issue.

However, now that you mentioned the Github settings, maybe it could be good to store them in project settings too, unless it makes sense to use these outside of GitSavvy?

Reply:
The `sublime-project` file might be shared within a team (though it is no likely), the `git config` is not. It might be one of the reasons why historically we were using `git config` rather than project settings.

Though, I agree that those information should not be stored at `git config` neither.